content/en/authors/yagut-shakizada/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+# Display name
+title: "Yagut Shakizada"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "yagut-shakizada"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: Research Student
+
+# Organizations/Affiliations
+organizations:
+- name: Nara Institute of Science and Technology
+  url: "http://www.naist.jp"
+
+# Short bio (displayed in user profile at end of posts)
+bio: Hi! It's Yagut from Azerbaijan. The research student of Software Design and Analysis Laboratory at Division of Information Science, NAIST. 
+
+# List each interest with a dash
+interests:
+- Cooking and tasting new foods
+- Exploring new places 
+- Watching traveling videos, drama movies and entertainment news
+
+
+education:
+  degree: Bachelor's Degree
+  courses:
+    - course: Bachelor's Degree in Process Automation Engineering
+      institution: Baku Higher Oil School
+      year: 2017-2022
+
+
+# Social/Academic Networking
+# For available icons, see: https://wowchemy.com/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:shakizada.yagut.ta2@naist.ac.jp'  
```

Comment:
I don't recommend listing your email address, as it can lead to receiving a lot of spam.